server/server.go

```diff
@@ -445,9 +446,15 @@ func (s *Server) Index(w http.ResponseWriter, _ *http.Request) {
 			LockPath:     lockURL.String(),
 			RepoFullName: v.Project.RepoFullName,
 			PullNum:      v.Pull.Num,
+			Path:         v.Project.Path,
+			Workspace:    v.Workspace,
 			Time:         v.Time,
 		})
 	}
+
+	//sort by date - newest to oldest
+	sort.SliceStable(lockResults, func(i, j int) bool { return lockResults[i].Time.After(lockResults[j].Time) })
```

Comment:
```suggestion
	sort.SliceStable(lockResults, func(i, j int) bool { return lockResults[i].Time.Before(lockResults[j].Time) })
```

It was sorting oldest to newest in my testing.

Reply:
I'm seeing that new is on top and old is on bottom with the `Time.After`:

![image](https://user-images.githubusercontent.com/23270849/64964178-8d47c480-d868-11e9-9e54-590787350c7b.png)

I also noticed the same in your screenshots regarding the GUI/sizing - did you have `Time.After` in place on those screenshots?

In case I misunderstood, did you want oldest to be on top?  

Thanks!

Reply:
I have no idea what I was looking at before 😕 you have it correctly.

One last thing I noticed is that the timestamps are kinda crazy. If we add a `TimeFormatted string` field to the `LockIndexData` struct and then set it here:

```go
	for id, v := range locks {
		lockURL, _ := s.Router.Get(LockViewRouteName).URL("id", url.QueryEscape(id))
		lockResults = append(lockResults, LockIndexData{
			// NOTE: must use .String() instead of .Path because we need the
			// query params as part of the lock URL.
			LockPath:      lockURL.String(),
			RepoFullName:  v.Project.RepoFullName,
			PullNum:       v.Pull.Num,
			Path:          v.Project.Path,
			Workspace:     v.Workspace,
			Time:          v.Time,
			TimeFormatted: v.Time.Format("02-01-2006 15:04:05"),
		})
	}
```

Then the times look like:
![image](https://user-images.githubusercontent.com/1034429/64972889-eb6faa00-d85e-11e9-9f12-0ed6314dca56.png)

Reply:
Thanks for all the feedback!  I made all of the changes requested, except for removing the `-` in the GUI (between the pull num and dir) to make things less crowded.  If you can take one more look I'll make any additional changes or proceed with merging.  👍